projects/plugins/boost/app/lib/critical-css/source-providers/providers/Singular_Post_Provider.php

```diff
@@ -142,4 +159,25 @@ public static function post_type_query( $post_type ) {
 	public static function get_success_ratio() {
 		return static::MIN_SUCCESS_URLS / static::MAX_URLS;
 	}
+
+	/**
+	 * Get post types that won't be included in the source providers list.
+	 *
+	 * @return array
+	 */
+	public static function get_ignored_post_types() {
+		/**
+		 * Filter the post types that won't be included in the source providers list.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param array $ignored_post_types An array of post types names.
+		 */
+		return apply_filters(
+			'jetpack_boost_critical_css_ignored_post_types',
+			array(
+				'e-landing-page', // Elementor's landing pages are broken. See https://github.com/elementor/elementor/issues/16244
+			)
+		);
+	}
```

Comment:
Compatibility code shouldn't be a part of our core codebase. This needs to move to the compatibility directory.

Reply:
Nice catch! I've updated this in acaddc5.